content/en/synthetics/troubleshooting/_index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Synthetics Troubleshooting
+kind: documentation
+description: Troubleshoot common Synthetics isses.
+beta: true
+further_reading:
+- link: "synthetics/"
+  tag: "Documentation"
+  text: "Manage your tests"
+- link: "synthetics/browser_tests"
+  tag: "Documentation"
+  text: "Configure a Browser Test"
+- link: "synthetics/api_tests"
+  tag: "Documentation"
+  text: "Configure an API Test"
+---
+
+If you experience issues setting up or configuring Datadog Synthetics, use this information to begin troubleshooting. If you continue to have trouble, [contact our awesome support team][1].
+
+## API Tests
+
+### Failed requests on a working endpoint
+
+You can sometimes get `5xx` or `4xx` when setting an API test on your endpoints. This happens most when hitting `Test URL` to perform a fast test, and then the error shows up in your regular test results. However, you know there's not actually an issue, because when you visit your website in a browser (or cURL it), you're getting `2xx`. 
```

Comment:
```suggestion
Sometimes you know you have a working endpoint because when you visit your website in a browser (or cURL it), you get `2xx` status code. But when setting an API test on this endpoints or when hitting `Test URL` to perform a fast test you get `5xx` or `4xx` status code.```